src/etc/natvis/libcore.natvis

```diff
@@ -6,34 +6,29 @@
       <Item Name="[ptr]">pointer</Item>
     </Expand>
   </Type>
+
   <Type Name="core::ptr::Shared&lt;*&gt;">
     <DisplayString>{{ Shared {pointer} }}</DisplayString>
     <Expand>
       <Item Name="[ptr]">pointer</Item>
     </Expand>
   </Type>
+
   <Type Name="core::option::Option&lt;*&gt;">
-    <DisplayString Condition="RUST$ENUM$DISR == 0x0">{{ None }}</DisplayString>
-    <DisplayString Condition="RUST$ENUM$DISR == 0x1">{{ Some {__0} }}</DisplayString>
+    <DisplayString Condition="RUST$ENUM$DISR == 0x0">None</DisplayString>
+    <DisplayString Condition="RUST$ENUM$DISR == 0x1">Some({__0})</DisplayString>
     <Expand>
-      <Item Name="[size]" ExcludeView="simple">(ULONG)(RUST$ENUM$DISR != 0)</Item>
+      <Item Name="[size]" ExcludeView="simple">(unsigned long)(RUST$ENUM$DISR != 0)</Item>
       <Item Name="[value]" ExcludeView="simple">__0</Item>
```

Comment:
What happens here in the `None` case?
Looks like it's going to print some garbage. Maybe it should be conditional?

Reply:
Agreed. I've added a condition to the `[value]` item, and removed the `[len]` item entirely.  `Option` doesn't have a `len` method, and I think synthesizing one just adds redundant noise to the debugger display.